Nodejs/Product/Nodejs/Logging/LiveLogger.cs

```diff
@@ -32,11 +32,19 @@ internal sealed class LiveLogger {
         private static volatile LiveLogger _instance;
         private static object _loggerLock = new object();
 
-        private NodejsDiagnosticsOptionsPage _diagnosticsOptions = null;
+        private NodejsDiagnosticsOptionsPage _diagnosticsOptions;
 
         private LiveLogger() {
-            if (NodejsPackage.Instance != null) {
-                _diagnosticsOptions = NodejsPackage.Instance.DiagnosticsOptionsPage;
+        }
+
+        private NodejsDiagnosticsOptionsPage DiagnosticsOptions {
+            get {
+                if (_diagnosticsOptions == null) {
+                    if (NodejsPackage.Instance != null) {

```

Comment:
simplify this check so that it's `if(_diagnosticsOptions == null && NodejsPackage.Instance != null)`